ci/playbooks/build_runner_image.yaml

```diff
@@ -36,4 +36,6 @@
         content: |
           make_edpm_deploy_env:
             DATAPLANE_RUNNER_IMG: "{{ ansibleee_runner_img }}"
+          make_edpm_compute_baremetal:
+            DATAPLANE_RUNNER_IMG: "{{ ansibleee_runner_img }}"
```

Comment:
I think we can set `ANSIBLEEE_IMAGE_URL_DEFAULT` and without any hacks in `install_yamls` it would be used. We can get rid of all the hacking for OPENSTACK_RUNNER_IMG from `install_yamls`.

[1] https://github.com/openstack-k8s-operators/openstack-ansibleee-operator/blob/main/api/v1alpha1/openstack_ansibleee_types.go#L201